src/atoms/Button/Button.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+import styled from 'styled-components';
+
+export interface ButtonProps {
+  text: string;
+  onClick?: () => void;
+  btnColor?: string;
+  className?: string;
+}
+
+export const Button: React.FC<ButtonProps> = ({ text, onClick, className }) => {
+  return (
+    <button
+      type="button"
+      onClick={onClick}
+      className={className}
+    >
+      { text }
+    </button>
+  )
+}
+
+export const StyledButton = styled(Button)`
```

Comment:
It is better to use const StyledButton = styled(button)`
and then use StyledButton inside Button Component.